mautrix_telegram/util/tgs_converter.py

```diff
@@ -0,0 +1,137 @@
+# Generated by Netbeans
+# Author: Eramde
+# Date: 09.2019
```

Comment:
This should have the normal AGPL license header like in other files (but with your name instead of mine)